pyhanko/pdf_utils/content.py

```diff
@@ -179,10 +184,12 @@ def __init__(
         resources: Optional[PdfResources] = None,
         box: Optional[BoxConstraints] = None,
         writer: Optional[BasePdfFileWriter] = None,
+        matrix: Optional[generic.ArrayObject] = None
```

Comment:
Something's up with the typing, either here or elsewhere in the PR. Please double-check.

Reply:
Changed them all to DictionaryObject